fr/README.md

```diff
@@ -0,0 +1,115 @@
+# Licences Creative Commons pour GitHub Projects
+
+Dans ce dépot vous trouverez des manières simples pour appliquer les licences Creatives Commons sur vos propres dépots Github en langage Markdown.
+
+> **ATTIENTION :**
+> Vous ne devez pas utiliser de licence Creative Commons licenses pour des logiciels.
+> Pour cela, préférez les licences Free Software telles que GPL, BSD, etc.
+> Les licences Creative Commons s’appliquent uniquement à la propriété intellectuelle.
+
+L’approche la plus simple pour trouver la bonne licence CC pour votre projet est le [site Creative Commons](https://creativecommons.org/choose/).
+
+Nous proposons ici trois de ces licences :
+
+* [Creative Commons Attribution 4.0 International](#cc-attribution-40-international)
+* [Creative Commons Attribution-ShareAlike 4.0 International](#cc-attribution-sharealike-40-international)
+* [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International](#cc-attribution-noncommercial-sharealike-40-international)
```

Comment:
Would you translate the license names as well. There are official translations for each one of them, you can find them in https://creativecommons.org/licenses/?lang=fr

These changes should also be applied on the following sections of the README.md.

Reply:
Done!